extensions-core/stats/src/test/java/org/apache/druid/query/aggregation/variance/sql/VarianceSqlAggregatorTest.java

```diff
@@ -724,4 +729,20 @@ public void testOverWindow()
         ))
         .run();
   }
+
+  @Test
+  public void testStddevNotSupportedOverWindow()
```

Comment:
I see why this test is here; I was looking for an agg in the `sql` module to move it there...

I've bumped into a line where finalize is set to [false here](https://github.com/apache/druid/blob/6cf6838eb974634ce5ae664d5ca877d8c804036a/sql/src/main/java/org/apache/druid/sql/calcite/rel/Windowing.java#L163) the comment is not entirely accurate; I think the window aggs do finalize now  [here](https://github.com/apache/druid/blob/6cf6838eb974634ce5ae664d5ca877d8c804036a/processing/src/main/java/org/apache/druid/query/rowsandcols/semantic/DefaultFramedOnHeapAggregatable.java#L315) so that comment should be removed or updated

Reply:
modified the comment which used to state we dont finalize